app/Config/core.php

```diff
@@ -113,25 +113,25 @@
  * will override the automatic detection of full base URL and can be
  * useful when generating links from the CLI (e.g. sending emails)
  */
-	//Configure::write('App.fullBaseUrl', 'http://example.com');
+    //Configure::write('App.fullBaseUrl', 'http://example.com');
 
 /**
  * Web path to the public images directory under webroot.
  * If not set defaults to 'img/'
  */
-	//Configure::write('App.imageBaseUrl', 'img/');
+    //Configure::write('App.imageBaseUrl', 'img/');
 
 /**
  * Web path to the CSS files directory under webroot.
  * If not set defaults to 'css/'
  */
-	//Configure::write('App.cssBaseUrl', 'css/');
+    //Configure::write('App.cssBaseUrl', 'css/');
 
 /**
  * Web path to the js files directory under webroot.
  * If not set defaults to 'js/'
  */
-	//Configure::write('App.jsBaseUrl', 'js/');
+    //Configure::write('App.jsBaseUrl', 'js/');
```

Comment:
__[staging][code_sniffer]__
* No space found before comment text; expected "// Configure::write('App.jsBaseUrl', 'js/');" but found "//Configure::write('App.jsBaseUrl', 'js/');"
* Inline comments must end in full-stops, exclamation marks, or question marks
* There must be no blank line following an inline comment